client/go/internal/cli/cmd/login.go

```diff
@@ -72,7 +72,15 @@ This command runs a browser-based authentication flow for the Vespa Cloud contro
 			})
 
 			if err != nil {
-				return fmt.Errorf("login error: %w", err)
+				switch err.Error() {
+				case "600":
+					cli.printErr(fmt.Errorf("Unable to login through social channels"),
+						"Your organization does not allow logging in through our social channels",
+						"Please login by entering your address into the email field")
+					return fmt.Errorf("Unauthorized")
```

Comment:
You can simply return `errHint(fmt.Errorf("Unauthorized"), "Your organization does not allow logging in through our social channels", "Please login by entering your address into the email field")` here, and it will be printed automatically.